aten/src/ATen/cpu/Utils.h

```diff
@@ -27,6 +27,9 @@ TORCH_API bool init_amx();
 // Detect if CPU supports Arm(R) architecture SVE ISA
 TORCH_API bool is_arm_sve_supported();
 
+// Detect if CPU supports Arm(R) architecture SVE ISA and BF16
+TORCH_API bool is_arm_sve_bf16_supported();
```

Comment:
Unrelated to this PR, but re-exporting cpuinfo APIs this way looks a bit like an anti-pattern...